libs/ui/src/lib/drag-and-drop-file-input/drag-and-drop-file-input.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { BehaviorSubject, Observable } from 'rxjs'
+import { distinctUntilChanged } from 'rxjs/operators'
+
+@Component({
+  selector: 'ui-drag-and-drop-file-input',
+  templateUrl: './drag-and-drop-file-input.component.html',
+  styleUrls: ['./drag-and-drop-file-input.component.css'],
+})
+export class DragAndDropFileInputComponent implements OnInit {
+  @Input() placeholder = 'Nom de mon fichier'
```

Comment:
oh do you speak french ? :)

Reply:
On top of the file do a 
```js
export const placeholder = 'dropFile'
marker('dropFile')
```
This will add en entry for the label to translate in i18n translation files.
And here use `placeholder` variable for the default value.
Then run the script for label extraction for i18n, check CONTRIBUTING.md please.